assets/src/edit-story/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -98,28 +105,27 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
     isMuting,
   ]);
 
+  const buttonText = useMemo(() => {
+    return isMuting
+      ? __('Muting video', 'web-stories')
+      : __('Mute video', 'web-stories');
```

Comment:
As per https://github.com/google/web-stories-wp/issues/8462#issuecomment-886803398, rename to "Remove audio" / "Removing audio"